envoy/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 ### Changes
 
 * [FEATURE] add newly-documented metrics. See #1326
+* [IMPROVEMENT] tags can now contain any characters. See #1404
```

Comment:
Is this accurate? We support unicode tags, afaik, but there are still some special characters we don't want in tags. (For example, `|` can break some stuff.)

Reply:
I'll be more specific, thanks!